lib/blpapi-wrapper.ts

```diff
@@ -373,11 +371,7 @@ export class Session extends events.EventEmitter {
                 //      until ES6 Map is available
                 this.subscriptions[cid] = s;
 
-                // Lookup by service the events we need to listen to.
-                if (!(serviceUri in SERVICE_TO_SUBSCRIPTION_EVENTS_MAP)) {
-                    throw new Error('Unknown service: ' + serviceUri);
-                }
-                var events: string[] = SERVICE_TO_SUBSCRIPTION_EVENTS_MAP[serviceUri];
+                var events = SERVICE_TO_SUBSCRIPTION_EVENTS_MAP[serviceUri];
```

Comment:
We need an `assert` here for the same reason for request/response, when the serviceUri is valid but we don't have a corresponding entry in the map.